core-common/src/main/java/org/zalando/nakadi/repository/db/StorageDbRepository.java

```diff
@@ -57,10 +70,12 @@ public StorageDbRepository(final JdbcTemplate template, final ObjectMapper mappe
     public Storage createStorage(final Storage storage) throws DuplicatedStorageException, RepositoryProblemException {
         try {
             jdbcTemplate.update(
-                    "INSERT INTO zn_data.storage (st_id, st_type, st_configuration) VALUES (?, ?, ?::jsonb)",
+                    "INSERT INTO zn_data.storage (st_id, st_type, st_configuration, st_default) " +
```

Comment:
Can't we use `STORAGE_FIELDS` here as well?

Reply:
Everywhere except this line we are not relying on the order of the fields. The main criteria was to have the set of fields to select the same as in RowMapper below. For insert operation it makes sense to see right here what is being inserted and in which order.